libr/main/rabin2.c

```diff
@@ -893,6 +893,13 @@ R_API int r_main_rabin2(int argc, const char **argv) {
 		return 1;
 	}
 	file = argv[opt.ind];
+
+	if (file && file[0] == '\0') {
+		eprintf ("Couldn't open empty path\n");
+		r_core_fini (&core);
```

Comment:
Why return here and goto below? Would be better to have a single exit point

Reply:
The `goto` below is in another file though.

Reply:
I was trying to follow existing conventions in `r_main_radare2` and `r_main_rabin2`